vault/resource_database_secret_backend_connection.go

```diff
@@ -315,6 +315,21 @@ func connectionStringResource() *schema.Resource {
 	}
 }
 
+func mysqlConnectionStringResource() *schema.Resource {
+	r := connectionStringResource()
+	r.Schema["tls_certificate_key"] = &schema.Schema{
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "x509 certificate for connecting to the database. This must be a PEM encoded version of the private key and the certificate combined.",
```

Comment:
Should this be marked private since it includes the private key?

```suggestion
		Description: "x509 certificate for connecting to the database. This must be a PEM encoded version of the private key and the certificate combined.",
		Sensitive:   true,
```